src/vscode-dts/vscode.proposed.contribTerminalQuickFixes.d.ts

```diff
@@ -3,4 +3,4 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// empty placeholder declaration for the `notebookPreload` contribution point
```

Comment:
one more question: did you mean to replace the proposed notebookStaticPreloads file?

Reply:
I created a PR https://github.com/microsoft/vscode/pull/164675 to bring `notebookStaticPreloads` file back, as missing it will make compile failed.

@andreamah Could you have a look if you have time? 🙏